coqbot.toml

```diff
@@ -0,0 +1,18 @@
+[bot]
+name="coqbot"
+email="coqbot@users.noreply.github.com"
+
+[server]
+domain="coqbot.heroku.com"
+
+[github]
+#webhook_secret = "xxx"
+#api_token = "xxx"
+
+[gitlab]
+#api_token = "xxx"
+
+[mappings]
+  [mappings.coq]
+  github = "coq/coq"
+  gitlab = "coq/coq"
```

Comment:
If you'd like to add some other mappings to demonstrate how you can do it for several repositories, you can additionally put the mappings for `coq/opam-coq-archive`.